Sources/NIOPosix/BSDSocketAPIPosix.swift

```diff
@@ -297,5 +297,35 @@ extension NIOBSDSocket {
         throw ChannelError.operationUnsupported
         #endif
     }
+
+    static func setUDPReceiveOffload(_ enabled: Bool, socket: NIOBSDSocket.Handle) throws {
+        #if os(Linux)
+        var isEnabled: CInt = enabled ? 1 : 0
```

Comment:
Does this need to be `var`?

Reply:
I think so: `option_value` in `setsockopt` takes an `UnsafeRawPointer`